docs/api/List/ListSubheader.md

```diff
@@ -0,0 +1,15 @@
+ListSubheader
+=======
+
+
+
+Props
+-----
+
+
+| Name | Type | Default | Description |
+|:-----|:-----|:-----|:-----|
+| children | node |  | The content of the ListSubheader. |
+| primary | bool | false | If true, the ListSubheader will have the theme primary color. |
```

Comment:
The docs is autogenerated with `npm run build:docs`. You have added the information at the wrong place 😬